src/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -201,7 +285,41 @@ To let the `FlutterViewController` present without an existing
 `FlutterViewController` without an engine reference.
 
 {% samplecode no-engine-vc %}
-{% sample Swift %}
+{% sample SwiftUI %}
+```swift
+import SwiftUI
+import Flutter
+
+struct ContentView: View {
+  // No EnvironmentObject needed.
```

Comment:
nit: remove? it may need environment object for other dependencies.

Reply:
Sounds good